tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "jsx": "react",
     "target": "esnext",
     "lib": ["dom", "dom.iterable", "esnext"],
+    "types": ["cypress", "@testing-library/cypress"],
```

Comment:
@alxnddr Since, you set up `tsconfig.json`, do you think there's any implication on adding typescript support for Cypress here?